bip-0340.mediawiki

```diff
@@ -110,7 +110,7 @@ The following conventions are used, with constants as defined for [https://www.s
 ** The function ''bytes(x)'', where ''x'' is an integer, returns the 32-byte encoding of ''x'', most significant byte first.
 ** The function ''bytes(P)'', where ''P'' is a point, returns ''bytes(x(P))''.
 ** The function ''int(x)'', where ''x'' is a 32-byte array, returns the 256-bit unsigned integer whose most significant byte first encoding is ''x''.
-** The function ''has_even_y(P)'', where ''P'' is a point, returns ''y(P) mod 2 = 0''.
+** The function ''has_even_y(P)'', where ''P'' is a point, returns ''not is_infinite(P) and y(P) mod 2 = 0''.
```

Comment:
We could also say something like "where P is a point and not is_infinite(P), returns y(P) ..." because this makes clear that the function should never actually have to handle the point at infinity and is more consistent with the reference implementation.

Reply:
Yes, I think that is slightly better.

Reply:
Done.